docs/userguide/tutorials/geometry/curves_and_surfaces/curve_closest_point.py

```diff
@@ -17,11 +17,11 @@
 # Visualisation
 # ==============================================================================
 
-Artist.clear()
+SceneObject.clear()
```

Comment:
this is wrong and there are many like this.
however, all files in this folder were copied from occ to be able to do some compatibility testing.
they are not meant to be part of the docs.
please remove...